src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -49,11 +48,61 @@ const CountdownContainer = styled.View`
   width: 100%;
 `
 
+const Bold = styled(Sans)`
+  font-weight: bold;
+`
 export class FairHeader extends React.Component<Props> {
+  getContextualDetails(artistsCount, firstArtist, lastArtist, exhibitorsCount, firstExhibitor, lastExhibitor) {
+    const {
+      node: { name: firstArtistName },
+    } = firstArtist
+    const {
+      node: { name: lastArtistName },
+    } = lastArtist
+    const {
+      exhibitors: [firstExhibitorName],
+    } = firstExhibitor
+    const {
+      exhibitors: [lastExhibitorName],
+    } = lastExhibitor
+
+    artistsCount = !firstArtistName ? artistsCount : artistsCount - 1
+    artistsCount = !lastArtistName ? artistsCount : artistsCount - 1
+    exhibitorsCount = !firstExhibitorName ? exhibitorsCount : exhibitorsCount - 1
+    exhibitorsCount = !lastExhibitorName ? exhibitorsCount : exhibitorsCount - 1
+
+    return (
+      <>
+        {(!firstArtistName && !lastArtistName) || !artistsCount ? null : (
+          <Flex flexDirection="row" flexWrap="wrap">
+            <Sans size="3">Works by </Sans>
+            <Bold size="3">{!firstArtistName ? null : firstArtistName + ", "}</Bold>
```

Comment:
Rather than using `font-weight: bold` (which would then go outside of the design system spec) this should be `Sans weight='medium'` (which is the "bold" of Sans). If you want to keep the `<Bold>` component tho can also do something like this:

```js
const Bold = styled(Sans).attrs({ weight: 'medium' })``
```
Where the `attrs` method allows one to pass specific props on through.